helm/ck-application/templates/mysql-statefulset.yaml

```
@@ -15,11 +15,12 @@ spec:
       labels: {{- include "ck-mysql.labels" . | nindent 8 }}
     spec:
       securityContext:
+        {{- include "ck-application.appArmorProfile.securityContext" (list .) | indent 8 }}
         fsGroup: {{ template "ck-application.fsGroup.coordinated" . }}
```

yamllint

[warning] wrong indentation: expected 6 but found 8

(indentation)